src/test/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPCTest.java

```diff
@@ -298,25 +297,6 @@ void testGetManagedBSSIDs() throws Exception {
 
 	@Test
 	@Order(2)
-	void testGetCurrentTxPower() throws Exception {
```

Comment:
Why is this test gone?

Reply:
method is gone - it doesn't make sense to have it anymore imo since the field is strongly typed now.